lib/file_validators/utils/content_type_detector.rb

```diff
@@ -48,7 +48,7 @@ def empty_file?
       alias :empty? :empty_file?
 
       def blank_name?
-        @filepath.nil? || @filepath.try(:empty?)
+        @filepath.nil? || @filepath.empty?
```

Comment:
I'm trying to keep the implementation as close to paperclip as possible
https://github.com/thoughtbot/paperclip/blob/master/lib/paperclip/content_type_detector.rb#L43
but yes...